config/default.yaml

```diff
@@ -53,6 +53,8 @@ executor:
             # k8s node selectors for approprate pod scheduling
             nodeSelectors: {}
             preferredNodeSelectors: {}
+            labels:
+              disk: 'screwdriver.cd/disk'
```

Comment:
Let's do set the default, if we set the default and ppl with other cluster didn't configure their k8s cluster correctly. Then the pod will be pending forever. We can leave it as empty and do a check in executor to do nothing if label doesn't exist.